### PR TITLE
Make macro be able to generate server side skeleton only

### PR DESCRIPTION
**Pull request by junha1**

### CODE REVIEW

File: remote-trait-object-macro/src/service.rs [hunk 1]
```diff
@@ -43,32 +43,42 @@ impl<T: Parse> Parse for SingleArg<T> {
 #[derive(Default)]
```
Comment (majecty) [inline remote-trait-object-macro/src/service.rs:43]: The commit message " Revise macro arguments " looks too general.
How about using a commit message that explains what you did like "Make macro be able to generate server side only".

File: remote-trait-object-macro/src/service.rs [hunk 2]
```diff
@@ -43,32 +43,42 @@ impl<T: Parse> Parse for SingleArg<T> {
...
-    pub remote_only: bool,
+    pub no_stub: bool,
+    pub no_skeleton: bool,
 }
 
 impl MacroArgsRaw {
     fn update(&mut self, ts: TokenStream2) -> syn::parse::Result<()> {
-        let x: SingleArg<TokenStream2> = syn::parse2(ts.clone())?;
+        if let Ok(x) = syn::parse2::<syn::Ident>(ts.clone()) {
+            if x == quote::format_ident!("no_stub") {
```
Comment (majecty) [inline remote-trait-object-macro/src/service.rs:59]: The variable name `x` is used in many places. What is the meaning that the `x` indicates?

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -12,7 +12,7 @@ jobs:
           fetch-depth: 1
```
Comment (majecty) [inline .github/workflows/test.yml:12]: Could you split this commit into another PR?

Comment (junha1) [reply .github/workflows/test.yml]: I omitted the commit

File: remote-trait-object-macro/src/service.rs (hunk 2 above)
Comment (junha1) [reply remote-trait-object-macro/src/service.rs]: I gave a meaningful name

File: remote-trait-object-macro/src/service.rs (hunk 1 above)
Comment (junha1) [reply remote-trait-object-macro/src/service.rs:43]: I updated

**Comment by majecty** [comment]
@junha1 Could you change the PR name like the commit title?
Other things are OK.
